pr-data.csv

```diff
@@ -1806,16 +1806,16 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,minifi/m
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors,org.apache.nifi.processors.hadoop.TestGetHDFSFileInfo.testRunWithPermissionsExceptionContent,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors,org.apache.nifi.processors.hadoop.TestGetHDFSFileInfo.testRecursiveGroupAllToAttributes,ID,Opened,https://github.com/apache/nifi/pull/6629,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors,org.apache.nifi.processors.hadoop.TestGetHDFSFileInfo.testRecursiveGroupDirToAttributes,ID,Opened,https://github.com/apache/nifi/pull/6629,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors,org.apache.nifi.processors.hadoop.TestGetHDFSFileInfo.testRunWithPermissionsExceptionContent,ID,Opened,https://github.com/apache/nifi/pull/6629,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-parquet-bundle/nifi-parquet-processors,org.apache.nifi.processors.parquet.TestConvertAvroToParquet.testData,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors,org.apache.nifi.processors.script.TestScriptedPartitionRecord.testWhenMultiplePartitions,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors,org.apache.nifi.processors.solr.TestQuerySolr.testAttributes,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors,org.apache.nifi.processors.splunk.TestPutSplunkHTTP.testHappyPathWithCustomQueryParameters,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors,org.apache.nifi.processors.splunk.TestPutSplunkHTTP.testHappyPathWithCustomQueryParametersFromFlowFile,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors,org.apache.nifi.processors.standard.QueryDatabaseTableRecordTest.testAddedRowsTwoTables,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors,org.apache.nifi.processors.standard.QueryDatabaseTableTest.testAddedRowsTwoTables,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors,org.apache.nifi.processors.standard.QueryDatabaseTableRecordTest.testAddedRowsTwoTables,ID,Opened,https://github.com/apache/nifi/pull/6525,																				
```

Comment:
Please follow up

Reply:
If you don't follow up soon, your PR may be closed/rejected, so you'll have another `Rejected`.